assignment4/sass/pages/_details.scss

```diff
@@ -0,0 +1,348 @@
+$report-bg-color : #ffffff;
+
+.middle-container {
+    width: 82%;
+    float: right;
+	.page-status {
+    	background-color: $report-bg-color;
+    	margin: 5px;
+    	padding-bottom:15px;
+    	padding-top:10px;
+		span {
+			margin-left:20px;
+		}
+	}
+	.report {
+		background-color: $report-bg-color;
+	    margin: 5px;
+	}
+	.test-report {
+		.header {
+    		padding-bottom:40px;
+			.header-text {
+				float: left;
+	    		width: 35%;
+	    		margin-left:20px;
+			}
+
+			h5 {
+				color:blue;
+			}
+
+			button {
+				float:left;
+	    		margin: 10px 20px 5px 20px;
+	    		background-color:#624b3d;
+	    		color:white;
+	    		padding:5px;
+	    		border-color:#624b3d;
+			}
+
+			.subscription {
+				float: right;
+			    margin-right: 30px;
+			    color: red;
+			    margin-top: 8px
+			}
+		}
+		.detail {
+		    width: 100%;
+    		.report-img {
+			    width: 45%;
+			    text-align:center;
+				img {
+			        width: 20%;
+				}
+    		}
+    		
+    	
+		}
+
+		.user-detail {
+			width: 50%;
+			float:right;
+			margin-top:-130px;
+			label {
+				color:blue;
+			}
+			input[type="text"] {
+			    border-left: none;
+			    border-right: none;
+			    border-top: none;
+			    display: block;
+			    margin-bottom: 20px;
+			    width: 91%;
+			    border-bottom: 2px solid;
+			    border-color: blue;
+    		}
+
+			button {
+				float: right;
+			    margin-right: 44px;
+			    background-color: #7E90e0;
+			    padding: 8px;
+			    border-color: #7e90e0;
+			    margin-top: -10px;
+			}
+			a {
+				text-decoration: none;
```

Comment:
@bhartijain Define  `text-decoration` `color`  for `a` tag when starting the project because by default a tag has color blue and underline. If you don't define it while starting the project everywhere you have to add it.